browser/brave_local_state_prefs.cc

```diff
@@ -126,6 +126,9 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   brave_vpn::RegisterLocalStatePrefs(registry);
   skus::RegisterLocalStatePrefs(registry);
 #endif
+#if BUILDFLAG(IS_WIN)
+  registry->RegisterStringPref(::prefs::kBraveVpnDnsConfig, std::string());
```

Comment:
this should be done in `brave_vpn::RegisterLocalStatePrefs`. We're using FindPreference so we don't care if this is not registered

Reply:
done